src/bundle/Resources/public/js/CKEditor/embed/content-inline/embed-inline-ui.js

```diff
@@ -17,6 +17,10 @@ class IbexaEmbedContentInlineUI extends IbexaEmbedBaseUI {
         return {
             contentId: items[0].ContentInfo.Content._id,
             contentName: items[0].ContentInfo.Content.TranslatedName,
+            locationId: items[0].id,
+            languageCodes: items[0].ContentInfo.Content.CurrentVersion.Version.VersionInfo.VersionTranslationInfo.Language.map(
```

Comment:
maybe put `items[0].ContentInfo.Content` into variable? it's also used in line 18 and 19, thanks to this variable you'll also shorten this line